content/blog/2020-09-17-delivering-learnr-tutorials-in-a-package/index.Rmarkdown

```diff
@@ -0,0 +1,353 @@
+
+
+
+### Getting your tutorial to your audience
+
+Your tutorial package has (finally!) been built, now it's time to get it out there!
+
+1. **[Commit](https://happygitwithr.com/rstudio-git-github.html#make-local-changes-save-commit) and [push](https://happygitwithr.com/rstudio-git-github.html#push-your-local-changes-online-to-github)** to GitHub to share your tutorial package with the world.
+
+1. **Have users *install* your package** from GitHub with: `devtools::install_github(“<your-repo>/<package-name>")`. (They will also need to install the learnr package.)
+
+    ```{r, eval = FALSE}
+    devtools::install_github("dcossyleon/tutorialpackage")
+    ```
+
+1. **Have users *run* your tutorial** by navigating to the *Tutorial pane* of their RStudio IDE and clicking *Start Tutorial*. 
+
+    * Alternatively, users could also run the following code to open the tutorial: `learnr::run_tutorial("<tutorial-name>", "<package-name>")`
+
+      ```{r, eval = FALSE}
+      learnr::run_tutorial("lesson1", "tutorialpackage")
+      ```
+
+We recommend including these latter two instructions in a README file, which we introduce below.
+
+
+
+### Create a README file
+
+You can include additional information in a `README.Rmd` file for your package. At a minimum, we recommend you include a line about how users can install and load the tutorials.
+
+1. **Run** `use_readme_rmd()` to create and open a README file.
+
+    ```{r, eval =  FALSE}
+    use_readme_rmd()
+    ```
+
+1. **Edit** the README to meet your needs. Need ideas of what to include? Check out [this resource](https://r-pkgs.org/whole-game.html#use_readme_rmd).
+
+1. **Click Knit** so that it creates a corresponding `README.md` file.
+
+    * Or, alternatively, run:
+
+    ```{r, eval = FALSE}
+    usethis::build_readme()
+    ```
+
+1. **Commit and push** your changes to GitHub.
+
+
+
+### Updating your package
+
+After your package has been installed by your users, you can continue to update or add tutorials. Each time you make changes to the contents of your package, you must *Build* then *Install and Restart* the package locally before pushing your changes to GitHub.
+
+Users can then access the updated content by simply re-installing your package.
+
+
+## That's a wrap!
+
+Congratulations, you've created a tutorial package!
+
+From here, there are many ways to extend what you've done. For example, you could decide to bundle up custom data sets in your package, as explained in the [R Packages](https://r-pkgs.org/data.html#data-sysdata) book. And you might consider adding [package documentation](https://r-pkgs.org/man.html#man-packages). It's also worth mentioning that if you are teaching using RStudio Cloud or RStudio Server,  you can install the tutorial package in your workspace or project as the instructor, and make the student experience even simpler --  all students would need to do is go to the website, push a button, and start the tutorial.
```

Comment:
```suggestion
From here, there are many ways to extend what you've done. For example, you could decide to bundle up custom data sets in your package, as explained in the [R Packages](https://r-pkgs.org/data.html#data-sysdata) book. And you might consider adding [package documentation](https://r-pkgs.org/man.html#man-packages). It's also worth mentioning that if you are teaching using RStudio Cloud or RStudio Server, you can install the tutorial package in your workspace or project as the instructor, and make the student experience even simpler --  all students would need to do is go to the website, push a button, and start the tutorial.
```

Reply:
I wasn't able to identify what the suggested change was here